Paintroid/src/main/res/layout/pocketpaint_layout_top_bar.xml

```diff
@@ -23,6 +23,7 @@
     android:id="@+id/pocketpaint_layout_top_bar"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:alpha="0.80"
```

Comment:
Might be 0.85 as the transparency shall be 15% if I'm not mistaken.

Reply:
this was discussed with wolfgang slany. we came to the conclusion that 0.8 is sufficient